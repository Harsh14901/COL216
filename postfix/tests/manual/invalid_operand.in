16-26*b-460+*72-7-*+
0a3*1-3890-45b3-***++
0511*3-f+2-6455---+*
5394l7356a-29---***++
50023*8-gr-+6357+*-+-
24-2+w7196+0e06+*--++
73*38+*3*8*4h215-*-*
03-6+u9-72*4**x2*60*-
842*-61+3+82u1-7*-*-
235-8-+59*1e9**+2+3+
