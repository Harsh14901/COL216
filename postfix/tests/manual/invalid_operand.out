[*] Executing test case - 16-26*b-460+*72-7-*+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 0a3*1-3890-45b3-***++
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 0511*3-f+2-6455---+*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 5394l7356a-29---***++
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 50023*8-gr-+6357+*-+-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 24-2+w7196+0e06+*--++
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 73*38+*3*8*4h215-*-*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 03-6+u9-72*4**x2*60*-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 842*-61+3+82u1-7*-*-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 235-8-+59*1e9**+2+3+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
